webpack.config.js

```diff
@@ -12,7 +12,8 @@ var wpconfig = {
     context: __dirname,
 
     entry: {
-        CourseOutline: './openedx/features/course_experience/static/course_experience/js/CourseOutline.js'
+        CourseOutline: './openedx/features/course_experience/static/course_experience/js/CourseOutline.js',
```

Comment:
Thanks, that helps, and it is an interesting idea. I don't think it will work in production because the two systems have completely separate static asset directories and are served with different paths on the CDN. It also will likely be an issue blocking the eventual goal of having Studio and the LMS be two separate IDAs. However, I'm fine with merging as is and revisiting it if this gets out of control.